ding/framework/middleware/functional/logger.py

```diff
@@ -45,4 +45,33 @@ def _logger(ctx: "OnlineRLContext"):
     return _logger
 
 
-# TODO offline logger
+def offline_logger(record_train_iter: bool = False) -> Callable:
```

Comment:
remove `record_train_iter` option, train_iter is the only x-axis of offline logger